spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java

```diff
@@ -161,7 +161,7 @@ public void testMGetCommand() {
 		byte[] value2 = "bar2".getBytes();
 		connection.set("foo1".getBytes(), value1);
 		connection.set("foo2".getBytes(), value2);
-		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String [] {"foo1", "foo2"}).build());
+		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String[] {"foo1", "foo2"}).build());
```

Comment:
Whitespaces just after/before curly brackets. Like it is here
https://github.com/spring-projects/spring-integration/pull/1756/files/9898963393caef111b8113fd82c899dcfa1f9b9d#diff-a678ddfa8edd4c1f9458bdc775af6e5aR247
exactly with this PR